src/lib/drupal/facilitySideNav.ts

```diff
@@ -8,25 +8,25 @@ const normalizeMenuItem = (item: MenuItem): SideNavItem => {
   if (item.items && item.items.length > 0) {
     item.items.forEach((i) => nestedItems.push(normalizeMenuItem(i)))
   }
-
   return {
     description: item.description,
     expanded: item.expanded,
     label: item.title,
     links: nestedItems,
     url: { path: item.url },
+    fieldMenuSection: item.field_menu_section || null,
   }
 }
 
 const normalizeMenuData = (menu: Menu): SideNavData => {
   // Bail early if no tree is provided
   if (!menu.tree || menu.tree.length === 0) return null
 
-  const links = []
+  let links = []
   menu.tree.forEach((item) => {
     return links.push(normalizeMenuItem(item))
   })
-
+  links = links.filter((link) => link.links.length > 0)
```

Comment:
I'm curious what items this is filtering out for Lovell compared to existing menus (that didn't seem to need this filtering)

Reply:
the lovell data comes with 2 extra top layer menu objects that are needed for some other purposes, but just muddy up the data for side navs.  This just filters those out

Reply:
thanks for clarifying, that makes sense. if you could add a short comment to that effect for our future selves, something like `// This filters out two lovell-specific menu items that aren't used in side navs.`

Reply:
Yep, moved that bit to ride along with the util functions, and added this comment for clairty